File: Ejercicio2.java
import java.util.Scanner;

public class Ejercicio2 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Ingrese el primer numero: ");
        double num1 = sc.nextDouble();
        System.out.print("Ingrese el segundo numero: ");
        double num2 = sc.nextDouble();
        System.out.print("Ingrese el tercer numero: ");
        double num3 = sc.nextDouble();

        double media = (num1 + num2 + num3) / 3;
        System.out.println("La media es: " + media);
    }
}

File: Ejercicio2.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ejercicio2
{
    internal class Program
    {
        static void Main(string[] args)
        {
            double x, y, z, a;
            Console.WriteLine("Ingrese el primer número: ");
            x=double.Parse(Console.ReadLine());
            Console.WriteLine("Ingrese el segundo número: ");
            y=double.Parse(Console.ReadLine());
            Console.WriteLine("Ingrese el tercer número: ");
            z=double.Parse(Console.ReadLine());
            a = (x + y + z) / 3;
            Console.WriteLine("El resultado de la media es: " + a);
        }
    }
}
